File: Programmers/Lv2/최댓값과 최소값/Ohjemin2.py
import java.util.*;
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] arr = s.split(" ");
        int max = 0;        
        for(int i=0;i<arr.length-1;i++){
            for(int j=0;j<arr.length-i-1;j++){
                if(Integer.parseInt(arr[j]) > Integer.parseInt(arr[j+1])){
                    String tmp = arr[j+1];
                    arr[j+1] = arr[j];
                    arr[j] = tmp;
                }
            }
        }
        return arr[0]+" "+arr[arr.length-1];
    }
}

File: Programmers/Lv1/자연수 뒤집어 배열로 만들기/Ohjemin2.java
import java.util.*;
class Solution {
    public int[] solution(long n) {        
        String sN = Long.toString(n);
        String[] arr = sN.split("");
        Stack<Integer> st = new Stack<>();
        for(int i=0;i<sN.length();i++){
            st.push(Integer.parseInt(arr[i]));
        }
        int size = st.size();
        int[] answer = new int[size];
        for(int i=0;i<size;i++){
            answer[i] = st.pop();
        }
        return answer;
    }
}
